### PR TITLE
Add option safari (and firefox?)

### PR DESCRIPTION
**Pull request by TheSlimvReal**
### Visible/Frontend Changes
- [x] There were some problems about adding a new option in safari which this solves.
- [x] Multi enum select is correctly reset on cancel

### Architectural/Backend Changes
--

### CODE REVIEW

File: src/app/core/configurable-enum/basic-autocomplete/basic-autocomplete.component.ts [hunk 1]
```diff
@@ -142,6 +143,7 @@ export class BasicAutocompleteComponent<O, V = O>
     let filteredOptions = this._options;
     this.showAddOption = false;
     clearTimeout(this.addOptionTimeout);
+    setTimeout(() => (this.showAddOption = false), 100);
```
Comment (sleidig) [inline src/app/core/configurable-enum/basic-autocomplete/basic-autocomplete.component.ts:146]: observing actual users add options, I am tending towards removing the delay for new options again. It was complex enough for a non-savvy user to figure out they can do this without things moving and changing. Maybe instead highlight the "Add new" option with different color a bit more than already?

Sorry, somewhat off-topic here actually. Just popped into my mind.

**Review by sleidig — APPROVED** [review]
Couldn't test for the target devices but code looks good :+1:

File: src/app/core/configurable-enum/basic-autocomplete/basic-autocomplete.component.ts (hunk 1 above)
Comment (TheSlimvReal) [reply src/app/core/configurable-enum/basic-autocomplete/basic-autocomplete.component.ts]: Got point. I removed the timeout and instead visually highlighted the add option text. Please have a look

Comment (sleidig) [reply src/app/core/configurable-enum/basic-autocomplete/basic-autocomplete.component.ts]: Nice. I like it.

In terms of UI logic, maybe we should use the blue accent color though (similar to add buttons above lists, etc.)?

Comment (TheSlimvReal) [reply src/app/core/configurable-enum/basic-autocomplete/basic-autocomplete.component.ts]: I get the point but it does not look good with the blue color.

**Pull request merged**